jep/0000/README.adoc

```diff
@@ -193,33 +193,34 @@ Additional whitelist entries can be specified by the file `$JENKINS_HOME/stapler
 
 == Motivation
 
-During exploratory testing, the Jenkins security team discovered a class of vulnerabilities in Jenkins.
-Common to all these vulnerabilities is the fact that Stapler does not make a distinction between views that are meant to be dispatched directly versus those meant to only be included in other views.
-This lack of distinction causes numerous problems in Jenkins where URLs can be constructed to directly access fragment views of models which oftentimes do not perform their own permission checks or other initialization logic.
-It was discovered through analysis of existing well-behaved views that they all had implicit or explicit use of page initialization logic including setting a response `Content-Type`.
-Most views would do this implicitly through the `l:layout` or `l:ajax` Jelly tags.
-Fragment views, on the other hand, do not normally use either of those tags, nor do they perform any page initialization logic of their own.
-Instead, fragments assume their environment has been properly set up to be included into a proper view.
-These assumptions usually included permission checks performed in the view itself to apply to its fragments.
-However, due to how Stapler dispatches URLs to views, fragments could sometimes leak data to unauthorized users.
+During exploratory testing, the Jenkins security team discovered a specific type of vulnerability spread throughout the ecosystem of Jenkins and Jenkins plugins.
+This vulnerability allows for many fragment views to be directly dispatched by Stapler using specifically crafted URLs.
+Stapler does not make a distinction between views that are meant to be dispatchable and those that are meant to only be included as fragment views.
+Fragment views rarely enforce permission checks of their own as they expect an implicit containing view to handle that.
+Some of these fragment views can leak information when dispatched directly, though many fragments will err early on due to referencing uninitialized variables.
```

Comment:
I'm not sure about the use of the word "err" here. It might be clearer to rephrase this sentence a litt.e